com.gitlab.sat_metalab.Splash.json

```diff
@@ -95,21 +95,20 @@
         {
             "name": "splash",
             "buildsystem": "simple",
-            "config-opts": [
-                "-DBUILD_GENERIC_ARCH=ON"
-            ],
             "build-commands": [
-                "./make_deps.sh",
+                "git submodule update --init --recursive",
+                "mkdir -p external/third_parties",
+                "./external/build_zmq.sh",
                 "cmake -DBUILD_GENERIC_ARCH=ON -DWITH_LTO=ON -DCMAKE_INSTALL_PREFIX=/app .",
```

Comment:
this could just be a `buildsystem: cmake-ninja`

Reply:
That's what I gathered from my research, I just changed and tested it locally and it passes.

Reply:
Hum indeed I could build ZMQ as a separate module and use `cmake-ninja` everywhere. I'll check whether this works.